src/types/Todo.ts

```diff
@@ -0,0 +1,6 @@
+export interface Todo {
+  id: number;
+  userId: number;
+  title: string;
+  completed: boolean | null;
```

Comment:
Consider changing the type of 'completed' property to 'boolean' only, instead of 'boolean | null', as a todo item should have a definite completed status (true or false).